package.json

```diff
@@ -37,5 +37,14 @@
     "linters": {
       "javascript": false
     }
-  }
+  },
+  "main": ".eslintrc.js",
```

Comment:
This line is causing deploy error. It can be safely removed, and this would resolve your issue.

Explanation:
In JavaScript applications, `main` property in `package.json` is used as reference to entry point - e.g., to file which should be launched first when running the app.
In this case `main` is not needed, because we don't use JavaScript in this task. And even if it was, `.eslintrc.js` should not be an entry point, as it's just a config file.

Reply:
Нажаль, не допомогло, я тоді спробую свторити нову гілку і зроблю проєкт ще раз